app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss

```diff
@@ -72,11 +72,17 @@ $govuk-modal-wide-breakpoint: $govuk-page-width + $govuk-modal-margin * 2 + $gov
   touch-action: none;
 }
 
-.app-o-template__body--modal {
+.gem-o-template--modal {
+  @include govuk-media-query($media-type: screen) {
+    overflow-y: inherit;
+  }
+}
+
+.gem-o-template__body--modal {
```

Comment:
I'm sure this used to be documented somewhere but I can't find it. Apologies - remind me what `gem-o` relates to, and why it isn't `gem-c`?

Reply:
Following the ITCSS model, I consider the `template` component being an object, not a component that's why I used a different prefix `gem-o`. Since we generally only use `gem-c` across our shared components, it would be good to question the need of type `-c-` prefix.